packages/immutadot/src/seq/ChainWrapper.js

```diff
@@ -45,7 +53,7 @@ class ChainWrapper {
    * @since 0.1.11
    */
   _absolutePath(path) {
-    return concat(toPath(this._path), toPath(path))
+    return [].concat(unsafeToPath(this._path), unsafeToPath(path))
```

Comment:
unsafeToPath is guaranted to return an array, and concat doesn't mutate the array, so you can do `unsafeToPath(this._path).concat(unsafeToPath(path))`